packages/nunchaku/nunchaku.0.5.1/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+maintainer: "simon.cruanes.2007@m4x.org"
+authors: ["Simon Cruanes" "Jasmin Blanchette"]
+homepage: "https://github.com/nunchaku-inria/nunchaku/"
+bug-reports: "https://github.com/nunchaku-inria/nunchaku/issues"
+dev-repo: "https://github.com/nunchaku-inria/nunchaku.git"
+build: [
+  ["jbuilder" "build" "-p" name "-j" jobs]
+  ["strip" "_build/default/src/main/nunchaku.exe"]
+]
+build-test: ["jbuilder" "runtest" "-j" jobs]
+build-doc: [make "doc"]
+depends: [
+  "ocamlfind" {build}
```

Comment:
Do you still require `ocamlfind` ? I don't see any use of `ocamlfind` or `Findlib` in your source code